.github/workflows/webpack.yml

```diff
@@ -0,0 +1,24 @@
+name: sentryConfig
+
+on:
+  release:
+    types: [published, prereleased]
```

Comment:
Are we going to create release tags in order to trigger the workflow? Otherwise, the workflow won't be started according to the documentation.

Didn't we want to update source maps every time we push to master? Like this
```
on:
  push:
    branches:
      - 'master'
```

Reply:
sending this up now